ONSdigital/census-rm-acceptance-tests-2021
# Sample file and print file to census spec

# Motivation and Context
Changes to test new Census printfile 

# What has changed
Sample Load code updated to new spec
Changes to which file is loaded
Changes to test new specification

# How to test?
Tests can be run locally or in GCP, will need related branches for actionsvc and actionexporter.
With new images running the tests should run as normal:  make test


# Links
https://github.com/ONSdigital/census-rm-actionexporter-service/pull/3
https://github.com/ONSdigital/census-rm-action-service/pull/3

# Screenshots (if appropriate):
